ui/main/src/app/store/states/feed.state.ts

```diff
@@ -9,20 +9,11 @@
 
 
 import {createEntityAdapter, EntityAdapter, EntityState} from '@ngrx/entity';
-import {readOrdinal, LightCard, severityOrdinal} from '@ofModel/light-card.model';
+import {LightCard} from '@ofModel/light-card.model';
 import {Filter} from '@ofModel/feed-filter.model';
 import {FilterService, FilterType} from '@ofServices/filter.service';
 
-/**
- * The Feed State consist of:
- *  * EntityState of LightCard
- *  * selectedCardId: the currently selected card id
- *  * lastCardLoaded the last card added / updated to the feed
- *  * message: last message during state processing
- *  * filters: a collection of filter to apply to the rendered feed
- *  * sortBySeverity: Indicates whether the cards in the feed should be sorted by severity before being sorted by
- *    publishDate (desc = latest first)
- */
+
 export interface CardFeedState extends EntityState<LightCard> {
```

Comment:
no just a remainder of the old code, i will remove

Reply:
in fact , it is necessary otherwise we need to implement custom methods to add/update/delete  lightcard in the list of lightcards.  I do not think we will gain that much performance implementing it custom.